data/pb/common-masters/FAQs.json

```diff
@@ -0,0 +1,64 @@
+{
+    "tenantId": "pb",
+    "moduleName": "common-masters",
+    "FAQs": [
```

Comment:
create the same file inside dss-dashboard 

https://github.com/egovernments/egov-mdms-data/tree/DEV/data/pb/dss-dashboard